mpc-core/src/ot/extension/kos15/sender.rs

```diff
@@ -86,46 +90,53 @@ fn encrypt_values<C: BlockCipher<BlockSize = U16> + BlockEncrypt>(
         let q = Block::from(q);
         let masks = [q.hash_tweak(cipher, j), (q ^ delta).hash_tweak(cipher, j)];
         if flip {
-            encrypted_values.push([input[0] ^ masks[1], input[1] ^ masks[0]]);
+            ciphertexts.push([input[0] ^ masks[1], input[1] ^ masks[0]]);
         } else {
-            encrypted_values.push([input[0] ^ masks[0], input[1] ^ masks[1]]);
+            ciphertexts.push([input[0] ^ masks[0], input[1] ^ masks[1]]);
         }
     }
-    encrypted_values
+    ciphertexts
 }
 
-impl ExtSenderCore {
+impl Kos15Sender {
     pub fn new(count: usize) -> Self {
         let mut rng = ChaCha12Rng::from_entropy();
+
+        let cointoss_share = rng.gen();
         let mut base_choice = vec![0u8; BASE_COUNT / 8];
         rng.fill_bytes(&mut base_choice);
+
         Self {
+            rng,
             cipher: Aes128::new_from_slice(&[0u8; 16]).unwrap(),
```

Comment:
There is no error in fixing the keys to all zeroes. It would be more explicit what we are doing here if we fixed it to sth more obvious like [1,2,3,4,5,6...]